slack-api-client/src/test/java/test_locally/api/methods/RequestFormBuilderTest.java

```diff
@@ -0,0 +1,81 @@
+package test_locally.api.methods;
+
+import com.slack.api.methods.RequestFormBuilder;
+import com.slack.api.model.block.LayoutBlock;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.junit.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class RequestFormBuilderTest {
+
+
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    private static class MyTestBlock implements LayoutBlock {
+        public static final String TYPE = "myTestBlock";
+        private final String type = TYPE;
+        private String blockId;
+    }
+
+    @Test
+    public void testGetJsonWithGsonAnonymInnerClassHandling() throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        // SETUP
+        Method method = RequestFormBuilder.class.getDeclaredMethod("getJsonWithGsonAnonymInnerClassHandling", List.class);
```

Comment:
You can test one of the public methods that internally calls getJsonWithGsonAnonymInnerClassHandling instead of trying to run the private method using reflection APIs.